frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -557,6 +565,32 @@ const applyFilters = <T, S>(
   )
 }
 
+function mergeProps(...props: any) {
+  const firstProps = props[0]
+  const restProps = props.slice(1)
+
+  if (!restProps.length) {
+    return mergeWith({}, firstProps)
+  }
```

Comment:
Seems strange to call `mergeWith` with no `customizer` function. From the original code, there is a comment mentioning not to mutate the original... so probably for this case and the one further down, we can just do `{ ...firstProps }`.